data/talkactions/scripts/online.lua

```diff
@@ -1,36 +1,22 @@
+local maxPlayersPerMessage = 10
+
 function onSay(player, words, param)
 	local hasAccess = player:getGroup():getAccess()
 	local players = Game.getPlayers()
-	local playerCount = Game.getPlayerCount()
-
-	player:sendTextMessage(MESSAGE_STATUS_CONSOLE_BLUE, playerCount .. " players online.")
+	local onlineList = {}
 
-	local i = 0
-	local msg = ""
 	for k, targetPlayer in ipairs(players) do
```

Comment:
You don't need the value of k anymore, use `for _, targetPlayer in ipairs(players)`